packages/contracts-bedrock/foundry.toml

```diff
@@ -23,6 +23,7 @@ extra_output = ['devdoc', 'userdoc', 'metadata', 'storageLayout']
 bytecode_hash = 'none'
 build_info = true
 build_info_path = 'artifacts/build-info'
+via_ir = true
```

Comment:
This is going to cause build times to be way too slow

Reply:
Ack. Just included it for debugging. Should've included a note as such.